src/pages/movie/[id].js

```diff
@@ -98,47 +139,63 @@ export default function Movie() {
 							backgroundRepeat: 'no-repeat, no-repeat',
 						}}
 					>
-						<Hl2>
+						<Hl1>
 							{objData.title || objData.name}
 							<span> ({objData.release_date.split('-')[0]})</span>
-						</Hl2>
+						</Hl1>
 						<Hl3>{objData.tagline}</Hl3>
-						<Hl4>
-							{'Release Date: '}
-							<span>{objData.release_date.split('-').reverse().join('.')}</span>
-							<br />
-							{' Genres: '}
-							<span>
-								{objData.genres.map((item, index) => {
-									return index ? ', ' + item.name : item.name;
-								})}
-							</span>
-							<br />
-							{' Runtime: '}
-							<span>{toHoursAndMinutes(objData.runtime)}</span>
-						</Hl4>
+						<Article>
+							<div>
+								<Hl4>
+									{'Release Date: '}
+									<span>
+										{objData.release_date.split('-').reverse().join('.')}
+									</span>
+									<br />
+									{' Genres: '}
+									<span>
+										{objData.genres.map((item, index) => {
+											return index ? ', ' + item.name : item.name;
+										})}
+									</span>
+									<br />
+									{' Runtime: '}
+									<span>{toHoursAndMinutes(objData.runtime)}</span>
+								</Hl4>
+							</div>
+							<BookmarkButton onClick={bookmarkHandler}>
+								<img src="../images/bookmark.svg" width="14px" height="14px" />
+								bookmark
+							</BookmarkButton>
+						</Article>
 						<Article>
 							<div>
 								<Picture
 									src={imagesBaseUrl + objData.poster_path}
 									alt={'image of ' + (objData.title || objData.name)}
 								/>
 							</div>
-							<div>
-								<Paragraph>{objData.overview}</Paragraph>
-							</div>
 						</Article>
-						{/* TrailerMenu */}
-						{trailerLoading && <p>loading...</p>}
-						{trailerError && <p>The content could not be loaded. Try again later.</p>}
-						{trailerData && trailerData.results.length > 0 && (
-							<TrailerMenu trailerData={trailerData} />
-						)}
+						<div>
+							<Paragraph>{objData.overview}</Paragraph>
+						</div>
+						<ButtonContainer>
+							{/* TrailerMenu */}
+							{trailerLoading && <p>loading...</p>}
+							{trailerError && (
+								<p>The content could not be loaded. Try again later.</p>
+							)}
+							{trailerData && trailerData.results.length > 0 && (
```

Comment:
A little suggestion:

```suggestion
							{trailerData && trailerData.results.length  && (
```

Reply:
Your suggestion leaves a '0' in the view if the array is empty.